docs/install/lockfile.md

```diff
@@ -46,3 +46,7 @@ print = "yarn"
 Bun v1.2 changed the default lockfile format to the text-based `bun.lock`. Existing binary `bun.lockb` lockfiles can be migrated to the new format by running `bun install --save-text-lockfile --frozen-lockfile --lockfile-only` and deleting `bun.lockb`.
 
 More information about the new lockfile format can be found on [our blogpost](https://bun.com/blog/bun-lock-text-lockfile).
+
+#### Automatic yarn.lock migration
+
+When running `bun install` in a project with a `yarn.lock` (v1) file but no `bun.lock`, Bun automatically migrates the lockfile to `bun.lock`. The original `yarn.lock` is preserved and can be removed manually after verification.
```

Comment:
this page is more about bun's lockfile itself. but if you want to mention migrations don't forget about the package-lock.json and now pnpm-lock.yaml